wordcount/count/FileOpenHandleAsciiCheatBenchmark.cs

```diff
@@ -0,0 +1,72 @@
+using System.Buffers;
+using System.Diagnostics;
+using BenchmarkData;
+
+namespace FileOpenHandleAsciiCheatBenchmark;
+
+public static class FileOpenHandleAsciiCheatBenchmark
+{
+    public static Count Count(string path)
+    {
+        const byte NEWLINE = (byte)'\n';
+        const byte SPACE = (byte)' ';
+        ReadOnlySpan<byte> searchValues = [SPACE, NEWLINE];
+
+        int wordCount = 0, lineCount = 0, byteCount = 0;
+        bool wasSpace = true;
+
+        byte[] buffer = ArrayPool<byte>.Shared.Rent(BenchmarkValues.Size);
+        using var handle = File.OpenHandle(path, FileMode.Open, FileAccess.Read, FileShare.Read, FileOptions.SequentialScan);
+
+        // Read content in chunks, in buffer, at count lenght, starting at byteCount
+        int count = 0;
+        while ((count = RandomAccess.Read(handle, buffer, byteCount)) > 0)
+        {
+            byteCount += count;
+            Span<byte> bytes = buffer.AsSpan(0, count);
+            
+            while (bytes.Length > 0)
+            {
+                // what's this character?
+                if (bytes[0] <= SPACE)
```

Comment:
You intend for any C0 control code to be treated as a space / newline?

Reply:
This code is intended to cheat as much as possible, assuming you have perfect knowledge of the documents you are reading. It won't be the primary benchmark shown. It's intended as a novelty.